modules/dataframe/include/inviwo/dataframe/io/csvreader.h

```diff
@@ -89,6 +89,20 @@ class IVW_MODULE_DATAFRAME_API CSVReader : public DataReaderType<DataFrame> {
     CSVReader& setNumberOfExampleRows(size_t rows);
     size_t getNumberOfExamplesRows() const;
 
+    /** 
+    * sets the string that indicates that a row should be commented/removed
+    * @see CSVReader::defaultRowComment 
+    */
+    CSVReader& setRowComment(std::string_view comment);
+    const std::string& getRowComment() const;
+
+    /** 
+    * sets the string that indicates that a row should be kept
```

Comment:
from  [ResearchDaniel](https://github.com/ResearchDaniel)
The functionality is a bit ambigous. For example: Does the row need to start with "only"? We might want to have a consistent naming for the comment/keep only. An alternative naming for these functions could be:
**includeRows(...), excludeRows(...)**
Suggest something in the lines of:
```
/*
 * Only parse lines starting with the specified characters. The \p startingWith character(s) will not be considered as values for the line.
 */
CSVReader& includeRows(std::string_view startingWith);
```